_layouts/default.html

```diff
@@ -15,7 +15,7 @@
     <link rel="stylesheet" href="{{ '/assets/css/ie.css' | relative_url }}">
     <![endif]-->
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
-
+    {% include head-custom.html %}
```

Comment:
With #31 merged, how would you feel about setting this up this way:

1. `_layouts/default.html` has `{% include head-custom.html %}`
2. `_includes/head-custom.html` has `{% include theme_color_meta_headers.html %}`

That way we can move custom headers into `head-custom.html` as a single entry point and use this across all the themes.

Reply:
@parkr Great idea! That will also clarify that this is meant to be customized if wanted, and it's certainly a good idea to use this across all themes. 
The latest force-push has added changed this accordingly :smile:.

Reply:
PS: If you prefer underscores as separators, of course I can change that for consistency, just let me know. 
I took inspiration from:
https://github.com/mmistakes/so-simple-theme/blob/master/_includes/head-custom.html
which uses the "kebab case" style with dash-separators, but snake-case is also fine with me :smile:

Reply:
@olifre Let's be consistent, whatever we choose.

Either: 

1. `head-custom.html` and `head-custom-theme-colors.html`, OR
2. `custom_headers.html` and `theme_color_meta_headers.html`

I like the kebab-style system since it's kind of a grouping (we use a head-custom prefix, so `head-custom-google-analytics.html` could be added later for example) and it seems easily readable to me.

Reply:
> I like the kebab-style system since it's kind of a grouping (we use a head-custom prefix, so `head-custom-google-analytics.html` could be added later for example) and it seems easily readable to me.

That's a great point, indeed!
I have pushed another commit doing the renaming, so now all is implemented according to the kebab-style variant :+1: .